src/pages/trust.js

```diff
@@ -0,0 +1,236 @@
+            TiDB Cloud is designed with strict security measures in all aspects
+            that are aligned with the market, so that you can focus on your
+            data, not protecting it.
+          </SecurityAndTrustCenterKeySecurityFeatures>
+        </div>
+        <SecurityAndTrustCenterCompliance reportIcon={AICPA_SOC_PNG} />
+        <GetStartedWithTiDBCloud />
+      </article>
+    </Layout>
+  )
+})
+
+const SecurityAndTrustCenterSummary = React.memo(({ children }) => {
+  const className = `SecurityAndTrustCenterSummary`
+  return <div className={className}>{children}</div>
+})
+
+const SecurityAndTrustCenterKeySecurityFeatures = React.memo(
+  ({ title, children, list }) => {
+    const className = `SecurityAndTrustCenterKeySecurityFeatures`
+    const classNameTitle = `${className}-title`
+    const classNameBody = `${className}-body`
+    const classNameList = `${className}-list`
+    return (
+      <div className={className}>
+        <div className={classNameTitle}>{title}</div>
+        <div className={classNameBody}>{children}</div>
+        <div className={`${classNameList} columns`}>
+          {list.map((item) => (
+            <SecurityAndTrustCenterKeySecurityFeaturesItem data={item} />
+          ))}
+        </div>
+      </div>
+    )
+  }
+)
+
+const SecurityAndTrustCenterKeySecurityFeaturesItem = React.memo(({ data }) => {
+  const { icon, title, list } = data
+
+  const className = `SecurityAndTrustCenterKeySecurityFeaturesItem`
+  const classNameIcon = `${className}-icon`
+  const classNameTitle = `${className}-title`
+  const classNameLine = `${className}-line`
+  const classNameList = `${className}-list`
+  return (
+    <div className={`${className} column`}>
+      <div className={classNameIcon}>
+        <img src={icon.publicURL} alt={classNameTitle} />
+      </div>
+      <div className={classNameTitle}>{title}</div>
+      <div className={classNameLine} />
+      <div className={classNameList}>
+        {Array.isArray(list) ? (
+          <ul>
+            {list.map((item) => (
+              <li>{item}</li>
+            ))}
+          </ul>
+        ) : (
+          list
+        )}
+      </div>
+    </div>
+  )
+})
+
+const SecurityAndTrustCenterCompliance = React.memo(({ reportIcon }) => {
+  // const { icon, title, list } = data
+  const className = `SecurityAndTrustCenterCompliance`
+  const classNameTitle = `${className}-title`
+  const classNameSummary = `${className}-summary`
+  const classNameReport = `${className}-report`
+  const classNameReportLeft = `${classNameReport}-left`
+  const classNameReportRight = `${classNameReport}-right`
+  const classNameReportRightHeader = `${classNameReportRight}-header`
+  const classNameReportRightHeaderTitle = `${classNameReportRightHeader}-title`
+  const classNameReportRightHeaderMore = `${classNameReportRightHeader}-more`
+  const classNameReportRightDescription = `${classNameReportRight}-description`
+  const classNameMore = `${className}-more`
```

Comment:
I don't think it's a concise way to define the classes like this. You didn't full use `classnames` package here,  so it makes the code look more complicated and unnecessary.  You can use `${className}-title` directly.